app/ui/src/main/java/de/mm20/launcher2/ui/launcher/search/SearchVM.kt

```diff
@@ -218,15 +232,6 @@ class SearchVM : ViewModel(), KoinComponent {
                     flowOf(emptyList())
                 }
                 val allApps = searchService.getAllApps()
```

Comment:
maybe it would make sense to set `previousResults` here as well. But which list is correct? There are three, after all. The normal one?